pages/_app.js

```diff
@@ -85,8 +89,14 @@ export default class HomePage extends App {
                 </nav>
               </Col>
             </Row>
-            <nav id="hiddenDesktopMenu" className="HiddenMenu HideMenu">
+            <nav id="hiddenDesktopMenu" className="HiddenMenu HideMenu ">
```

Comment:
Why the style of fonts were changed?
![image](https://github.com/catho/ssr-quantum-simulator/assets/3389749/dbb250cb-7592-4bf8-831a-56bf814dcc8f)

Reply:
fixed